browser/ui/views/split_view/split_view.cc

```diff
@@ -77,13 +77,16 @@ SplitView::SplitView(Browser& browser,
       std::make_unique<views::WebView>(browser_->profile()));
   secondary_contents_web_view_ = secondary_contents_container_->AddChildView(
       std::make_unique<ActivatableContentsWebView>(browser_->profile()));
+  secondary_contents_scrim_view_ = secondary_contents_container_->AddChildView(
```

Comment:
we need tests for this in splitview to make sure the correct view is darkened

Reply:
Added test here https://github.com/brave/brave-core/pull/27172/commits/8c023c0bbbf88ab00f411a1ab9beefb799453cbe

Reply:
I spoke with @simonhong about this last night. Some additional changes must be made to split view to fully support the tab-modal scrim view as there are two "tabs" in split view mode (active and inactive) and we're only showing the scrim on the active tab at the moment. Since the scrim view feature flags are disabled at the moment, we decided to enter a follow-up issue to fix this in split view post-cr134 merge here https://github.com/brave/brave-browser/issues/43947.